pkg/test/common.go

```diff
@@ -29,6 +29,20 @@ const (
 	SleepTime     = 100 * time.Millisecond
 )
 
+//nolint:gochecknoglobals
+var TestBlobDigests = map[string]map[int]string{
+	"zot-cve-test": {
+		0: "7a0437f04f83f084b7ed68ad9c4a4947e12fc4e1b006b38129bac89114ec3621",
```

Comment:
Can we avoid using the digest values?
The issue is different versions of skopeo copy the image with different hashes.
So different developers may have different values on their environments.